lib/logstash/runner.rb

```diff
@@ -63,15 +44,6 @@ def run(args)
         end
         return LogStash::Agent.run($0, agent_args)
       end,
-      "rspec" => lambda do
```

Comment:
We never spoke to clean this option. We added the ```bin/rspec``` as complement, are we sure we want to get rid of this and move to bin/rspec ?